src/app/cart/cart.tpl.html

```diff
@@ -29,7 +29,7 @@ <h3 class="panel-name" translate>Your Gift Cart</h3>
                   <tbody>
                     <tr class="giftsum-gift-row" ng-repeat="i in $ctrl.cartData.items">
                       <td class="td-gift">
-                        <img src="http://placehold.it/90x70" class="giftsum-profile pull-left">
+                        <img ng-src="{{$ctrl.imgDomainDesignation}}/bin/crugive/image?thumbnailDesigNumber={{i.designationNumber}}" class="giftsum-profile pull-left">
```

Comment:
I think it'd make sense to make this a component <designation-img designation=""/>, so we aren't duplicating the url configuration everywhere it's used.